src/applications/facility-locator/config.js

```diff
@@ -2,6 +2,11 @@
 import environment from '../../platform/utilities/environment';
 import isProduction from '../../platform/utilities/environment/isProduction';
```

Comment:
There is an outstanding PR that would require a change here (https://github.com/department-of-veterans-affairs/vets-website/pull/9162), so heads up.

Reply:
Ooo, I like.